### PR TITLE
GSoC: Add proposals for Keptn

### PR DESCRIPTION
**Pull request by thschue**

### CODE REVIEW

File: summerofcode/2023.md [hunk 1]
```diff
@@ -244,3 +244,33 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
...
+- Expected project size: 175h
+- Difficulty: Easy
+- Upstream Issue (URL): https://github.com/keptn/lifecycle-toolkit/issues/745
+
+#### Argo Rollouts integration for the Lifecycle Toolkit
+- Description: Currently, Keptn (LT) supports Kubernetes Deployments, StatefulSets and Daemonsets for the usage of Pre-/Post-Deployment Tasks and Evaluations. Although these resources might fit for many use cases, Argo Rollouts are also evolving and it would be great to support them as well. The goal of this project is to add Argo Rollouts as a new resource type for the LT.
+- Expected outcome: The Keptn (LT) supports Argo Rollouts as a new resource type for the usage of Pre-/Post-Deployment Tasks and Evaluations. Therefore, if a Rollout is used, the Deployment is held back until all Pre-Deployment Tasks are finished and the evaluation is passed. After the Deployment is finished, the Post-Deployment Tasks and Evaluations are executed.
+- Recommended Skills: Go, Kubernetes, Argo Rollouts
+- Mentor(s): Thomas Schuetz @thschue (thomas.schuetz at t-sc dot eu), Giovanni Liva @giovanniliva (giovanni.liva at dynatrace dot com)
```
Comment (thisthat) [inline summerofcode/2023.md:264]:
```suggestion
- Mentor(s): Thomas Schuetz @thschue (thomas.schuetz at t-sc dot eu), Giovanni Liva @thisthat (giovanni.liva at dynatrace dot com)
```

Comment (thschue) [reply summerofcode/2023.md]: fixed this

**Comment by prakrit55** [comment]
hey @thschue  , I think two more proposals have to be resolved "Keptn Plugin for Backstage" and "Revamp the content and the aesthetics of the keptn.sh website"

**Review by aliok — COMMENTED** [review]
LGTM

Mentors are in https://github.com/cncf/foundation/blob/main/project-maintainers.csv and the content of the ideas are good.

Thanks!

**Comment by aliok** [comment]
> "Revamp the content and the aesthetics of the keptn.sh website"

I don't want to get in between, but documentation only tasks are not applicable for GSoC.

**Comment by thschue** [comment]
> > "Revamp the content and the aesthetics of the keptn.sh website"
> 
> I don't want to get in between, but documentation only tasks are not applicable for GSoC.

Where did you find this? I think this is not part of this PR ...

**Comment by aliok** [comment]
@thschue  It was mentioned in this comment: https://github.com/cncf/mentoring/pull/836#issuecomment-1433007532

**Comment by thschue** [comment]
> hey @thschue  , I think two more proposals have to be resolved "Keptn Plugin for Backstage" and "Revamp the content and the aesthetics of the keptn.sh website"

The Backstage Plugin has already been merged and the content thing is not in scope of GSoC.

**Comment by aliok** [comment]
LGTM

**Review by thisthat — APPROVED** [review]

**Review by nate-double-u — APPROVED** [review]
Thanks for this @thschue

**Pull request merged**